composer.json

```diff
@@ -27,37 +28,37 @@
   ],
   "homepage": "http://www.taotesting.com",
   "require": {
-    "oat-sa/generis": "13.10.1",
-    "oat-sa/tao-core": "46.8.5",
+    "oat-sa/extension-pcisample": "2.10.2",
+    "oat-sa/extension-tao-backoffice": "5.1.0",
+    "oat-sa/extension-tao-clientdiag": "7.8.1",
     "oat-sa/extension-tao-community": "9.1.0",
-    "oat-sa/extension-tao-funcacl": "6.0.1",
     "oat-sa/extension-tao-dac-simple": "6.8.3",
-    "oat-sa/extension-tao-testtaker": "7.7.3",
+    "oat-sa/extension-tao-delivery": "14.19.0",
+    "oat-sa/extension-tao-delivery-rdf": "13.1.0",
+    "oat-sa/extension-tao-eventlog": "2.10.2",
+    "oat-sa/extension-tao-funcacl": "6.0.1",
     "oat-sa/extension-tao-group": "6.6.2",
     "oat-sa/extension-tao-item": "10.17.0",
+    "oat-sa/extension-tao-itemqti": "25.18.1.1",
```

Comment:
This is the only real change. The other lines are affected because of the config "sort-packages".